### PR TITLE
Persistable playlists

### LINKED ISSUE #1137: FR: Persisted playlist (and playlist url)
Our Grafana dashboards are becoming quite popular with our Ops groups. I would like to be able to define different play lists, categorized by arbitrary subject areas so each group can play the dashboards relevant to them.

### PR DESCRIPTION
**Pull request by bergquist**
Lets have the discussion about the persistable playlists here. @utkarshcmu @nikita-graf

closes #1137

### CODE REVIEW

**Comment by utkarshcmu** [comment]
Looks good to me... except what @nikita-graf said about the sorting arrows in #3649. We need to fix that before merging this to master. Rest everything looks awesome :)

**Comment by utkarshcmu** [comment]
@bergquist - 2 more improvements : #3664 & #3665 , if they make sense then please merge them. :)

**Comment by utkarshcmu** [comment]
So, whats left in this PR from getting merged to master?

**Comment by bergquist** [comment]
@utkarshcmu @nikita-graf Seems like one of you have not signed the CLA. Please do so at https://cla-assistant.io/grafana/grafana?pullRequest=3655

**Comment by nikita-graf** [comment]
Just signed it.

**Comment by utkarshcmu** [comment]
Updated docs #3690

File: pkg/api/api.go [hunk 1]
```diff
@@ -169,6 +172,17 @@ func Register(r *macaron.Macaron) {
 			r.Get("/tags", GetDashboardTags)
 		})
 
+		// Playlist
+		r.Group("/playlists", func() {
+			r.Get("/", wrap(SearchPlaylists))
+			r.Get("/:id", ValidateOrgPlaylist, wrap(GetPlaylist))
+			r.Get("/:id/playlistitems", ValidateOrgPlaylist, wrap(GetPlaylistItems))
```
Comment (torkelo) [inline pkg/api/api.go]: är inte `/playlists/1232/items`  enklare, vad är skillanden mellan items och dashboards?

**Pull request merged**